datadog_checks_base/datadog_checks/base/checks/kubelet_base/base.py

```diff
@@ -0,0 +1,199 @@
+# (C) Datadog, Inc. 2020
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import json
+from datetime import datetime, timedelta
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.base.utils.date import UTC, parse_rfc3339
```

Comment:
Let's use relative throughout:

```suggestion
from ...utils.date import UTC, parse_rfc3339
from .. import AgentCheck
```